src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -180,6 +180,12 @@ $(function () {
 
     $("body").removeClass("nojs");
 
+    var scrollpos = localStorage.getItem('scrollpos');
```

Comment:
Can you check if we might want to use sessionStorage instead of localStorage? This would e.g. prevent race conditions if I have multiple tabs open:
https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage

Reply:
That is a good idea! Although a race-condition might be very rare here, as the scrollpos is saved to localStorage just before leaving the page and cleared onDOMReady for the new page. So one would need to be quick switching tabs – or at least quickly abort loading and then switch tab. But nonetheless, sessionStorage is the way to go here! Thanks!